traitcuration/traits/views.py

```diff
@@ -0,0 +1,33 @@
+from django.shortcuts import render, get_object_or_404
+from .models import Trait
+
+
+def browse(request):
+    traits = Trait.objects.all()
+    status_dict = get_status_dict(traits)
+    context = {"traits": traits, "status_dict": status_dict}
+    return render(request, 'traits/browse.html', context)
+
+
+def trait_detail(request, pk):
+    trait = get_object_or_404(Trait, pk=pk)
+    context = {"trait": trait}
+    return render(request, 'traits/trait_detail.html', context)
+
+
+def get_status_dict(traits):
```

Comment:
So concerning the `views.py` module, do you suppose it would be better to leave only the actual views in it (e.g. something which returns `render(...)`, and move helper routines, such as `get_status_dict`, into a separate module?

Reply:
Yeah, I also thought about it and it would be cleaner. I moved it into a `utils.py` module